Automattic/pocket-casts-ios
# Tracks: Adds analytics to the Apple Watch and Files settings

| 📘 Project: #154 | 🛫 Depends on: #313  |
|:---:|:---:|

- `settings_apple_watch_shown`: When the watch settings are displayed to the user
- `settings_apple_watch_auto_download_up_next_toggled`: When the user toggles the Auto Download Up Next option
- `settings_apple_watch_auto_download_episodes_changed`: When the user changes the Number of Episodes option
- `settings_apple_watch_auto_download_delete_downloads_toggled`: Whent the user toggles the Delete Downloads Outside of Limit option
- `settings_files_shown`: When the files settings are displayed to the user
- `settings_files_auto_add_up_next_toggled`: When the user toggles the Auto Add to Up Next option
- `settings_files_delete_local_file_after_playing_toggled`: When the user toggles the Delete Local File option
- `settings_files_delete_cloud_file_after_playing_toggled`: When the user toggles the Delete Cloud File option
- `settings_files_auto_upload_to_cloud_toggled`: When the user toggles the Auto Upload to Cloud option
- `settings_files_auto_download_from_cloud_toggled`: When the user toggles the Auto Download From Cloud option
- `settings_files_only_on_wifi_toggled`: When the user toggles the Only on Wifi option

## To test

1. Go to the profile tab
2. Tap on the cog
3. Tap on Apple Watch
4. ✅ `🔵 Tracked: settings_apple_watch_shown`
9. Go through each setting and change its value
12. Check console and verify each setting has a corresponding event in console and the value displayed is the value that you selected
13. If the option changed is a switch then the event should be suffixed `_toggle` if it's a value change it should be suffixed with `_changed`
14. Repeat the steps on the Files settings
15. Verify the ✅ `🔵 Tracked: settings_files_shown` event is triggered when it appears


## Checklist
